esp/platform/espcontext.cpp

```diff
@@ -929,27 +975,40 @@ unsigned getSlowProcessingTime()
     return false;
 }
 
+//#ifndef _CONTAINERIZED
+void NEWESPLOG(const char* fmt, ...)
+{
+    va_list args;
+    va_start(args,fmt);
+    VALOG(getEspLogCategory(nullptr), unknownJob, fmt, args);
+    va_end(args);
+}
+
 void ESPLOG(LogLevel level, const char* fmt, ...)
 {
-    if (getEspLogLevel(NULL)>=level)
-    {
-        va_list args;
-        va_start(args,fmt);
-        VALOG(MCdebugInfo, unknownJob, fmt, args);
-        va_end(args);
-    }
+    va_list args;
+    va_start(args,fmt);
+    VALOG(getEspLogCategory(nullptr), unknownJob, fmt, args);
```

Comment:
Is this correct? By ignoring the level passed to the function, aren't we effectively giving every message the same log level (i.e., the level assigned to the process)? If the process is configured with LogMin, then all messages will be logged if any message is logged.

Reply:
yep, thanks for pointing out, will restore, but ultimately ESPLOG will go away under this proposal (at least this version of it)